spec/unit/files/servicenow_spec.rb

```diff
@@ -188,10 +199,40 @@
         end
 
         it 'decrypts the password' do
-          expect(ServiceNowRequest).to receive(:new).with(anything, anything, anything, 'admin', 'password')
+          expect(ServiceNowRequest).to receive(:new).with(anything, anything, anything, 'admin', 'password', 'oauth_token')
           expect { servicenow('example.puppet.com') }.to raise_error(NoMethodError)
         end
       end
     end
+
+    context 'with hiera-eyaml encrypted oauth_token' do
```

Comment:
Similar comment to https://github.com/puppetlabs/puppetlabs-servicenow_cmdb_integration/pull/25/files#r444431325 re: this also testing "oauth token works".